### PR TITLE
Added new functions of geoip 1.1.0 WI-45203

### PR DESCRIPTION
**Pull request by aalwash**
YouTrack issue: https://youtrack.jetbrains.com/issue/WI-45203

### CODE REVIEW

File: meta/.phpstorm.meta.php [hunk 1]
```diff
@@ -413,6 +413,7 @@ function argumentsSet($setName, $values) {
     expectedArguments(\debug_print_backtrace(), 0, DEBUG_BACKTRACE_IGNORE_ARGS);
     expectedArguments(\debug_backtrace(), 0, DEBUG_BACKTRACE_PROVIDE_OBJECT|DEBUG_BACKTRACE_IGNORE_ARGS);
 
+    expectedArguments(\geoip_database_info(), 0, GEOIP_COUNTRY_EDITION);
```
Comment (wbars) [inline meta/.phpstorm.meta.php]: According to the [docs](http://php.net/manual/en/function.geoip-database-info.php) the function expect several constants, not just one. Could you please add them as well? 
> You can use the various constants defined with this extension (ie: GEOIP_*_EDITION).

Comment (aalwash) [reply meta/.phpstorm.meta.php]: Thanks for notifying, I will fix this asap and make a new PR

I did a upstream master merge, but accidentally merged the wrong commits
So this PR is broken, will make a new one tonight

Comment (aalwash) [reply meta/.phpstorm.meta.php]: Fix the PR and reverted to upstream master changes + my geoip meta changes

Let me know if you're OK with this.

**Comment by wbars** [comment]
Thank you.

**Pull request merged**